app/views/headerFooterTemplate/Head.scala.html

```diff
@@ -14,10 +14,21 @@
  * limitations under the License.
  *@
 
+@import config.FrontendAppConfig
+
 @this()
 
-@(headBlock: Option[Html] = None)
+@(headBlock: Option[Html] = None)(implicit appConfig: FrontendAppConfig)
+
 @headBlock
 <!--[if lte IE 8]><script src='@controllers.routes.Assets.versioned("javascripts/html5shiv.min.js")'></script><![endif]-->
 <!--[if lte IE 8]><link href='@controllers.routes.Assets.versioned("stylesheets/application-ie-8.css")' rel="stylesheet" type="text/css" /><![endif]-->
 <!--[if gt IE 8]><!--><link href='@controllers.routes.Assets.versioned("stylesheets/application.css")' media="screen" rel="stylesheet" type="text/css" /><!--<![endif]-->
+
+@if(appConfig.googleTagManagerId != "N/A") {
+  <script type="text/javascript" src='@controllers.routes.Assets.versioned("javascripts/google-tag-manager.js")'></script>
```

Comment:
this is not picking up the file 

https://github.com/hmrc/charities-frontend/commit/8364074094604fbf81109933066c4b367ff8f19d
the javascript file is in public in this example

Reply:
It's worth noting that these services are set up slightly different, so the way it works will be a little different.

Reply:
If there are still issues afterwards, here's DDCNLS's guide which might help. The implementations are slightly different but all the code should be the same. https://confluence.tools.tax.service.gov.uk/display/DDCNLS/Adding+GTM+to+services